Assets/MirageXR/Player/Scripts/Mobile/Popup/LoginView.cs

```diff
@@ -113,6 +115,10 @@ private void OnClickRegister()
     {
         Application.OpenURL(DBManager.registerPage);
     }
+    private void OnClickDeleteAccount()
+    {
+        Application.OpenURL(DBManager.deleteAccount);
```

Comment:
It won't pass Apple's inspection. We need to delete the account from the app, not open the page in the browser. If we don't have access to the API, we can open the page in webview directly in the app. but we have to open the page with the user already logged in.